Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0-dev
+version: 0.3.1
```

Comment:
If it's necessary to specify a different tag for the dev branch, it should be 0.3.0-dev1 or similar

Reply:
I've changed it back to 0.3.0-dev.